libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxSpeechApi.kt

```diff
@@ -17,17 +17,23 @@ import kotlinx.coroutines.launch
  * @property context Context
  * @property accessToken String
  * @property language [Locale] language (ISO 639)
+ * @property baseUri base URL (optional)
  */
-class MapboxSpeechApi(
+class MapboxSpeechApi @JvmOverloads constructor(
     private val context: Context,
     private val accessToken: String,
-    private val language: String
+    private val language: String,
+    private val baseUri: String? = null
```

Comment:
`MapboxSpeechApiOptions`?

Reply:
Going to address in a separate PR as a follow up.